src/components/Activities/ActivitiesList.tsx

```diff
@@ -0,0 +1,65 @@
+import { Activities } from "./Definition";
+import ListView from "./ListView";
+
+const builtInActivities: Activities[] = [
+  {
+    id: 1,
+    name: "Kayaking",
+    description: "Relax on a nice sunny day with a trip down the river.",
+    voteCount: 4,
+    category: "outdoor",
+  },
+
+  {
+    id: 2,
+    name: "Movies",
+    description:
+      "If the weather is bad outside, enjoy a cozy movie in a movie theater.",
+    voteCount: 2,
+    category: "indoor",
+  },
+
+  {
+    id: 3,
+    name: "Fondue",
+    description:
+      "If you enjoy cooking, try making fondue as a romantic activity.",
+    voteCount: 2,
+    category: "indoor",
+  },
+
+  {
+    id: 4,
+    name: "Kayaking",
+    description: "An outdoor activity your friends will enjoy.",
+    voteCount: 6,
+    category: "outdoor",
+  },
+
+  {
+    id: 5,
+    name: "Salsa Class",
+    description:
+      "Learn a new dance and meet new people by attending a salsa class.",
+    voteCount: 4,
+    category: "indoor",
+  },
+];
+
+const ActivityList = () => {
+  return (
+    <ListView
```

Comment:
Ok I figured out how to get the generic working fully so we could remove all idea of the Activity from the `ListView`:

Step 1: we need to tell the component what type to treat as T any time we render it:

```suggestion
    <ListView<Activities>
```

Reply:
Done in 6f5ba69